automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppLifecycleSpec.scala

```diff
@@ -53,12 +53,12 @@ class AppLifecycleSpec
     ("create CROMWELL app, delete it and re-create it with same disk",
      createAppRequest(AppType.Cromwell, "cromwell-test-workspace", None),
      false,
-     true
+     false
     ),
     ("create GALAXY app, start/stop, delete it and re-create it with same disk",
      createAppRequest(AppType.Galaxy, "Galaxy-Workshop-ASHG_2020_GWAS_Demo", None),
-     true,
-     true
+     false,
```

Comment:
Disabling testing stop/start and PD for Galaxy. Technically we do support these things but it makes the tests long and flaky.

Also not sure testing stop/start is good while running tests in parallel reusing the same cluster -- stopping an app resizes the nodepool so can affect other apps running in the cluster.

Reply:
I understand, but this test actually prevented us from shipping a buggy version of galaxy like 2 weeks ago

Reply:
Maybe we enable PD tests but disable stop/start?